### ISSUE TITLE
It is only stuck , this is what it shows. Please help.

### ISSUE THREAD

**Issue by Jay59376** [toolbox4minecraft/amidst#1040]
<img width="995" alt="image" src="https://user-images.githubusercontent.com/68957356/129313697-dd28a4e5-40e5-4e2b-a028-e63d6b6a3877.png">

**Comment by Chen-Rain** [comment]
Well, it seems that you didn't create a world yet. To check the uncreated world, you can click "File" on the menubar (just beside the Apple icon and Amidst name), then click "New From Seed" or "New From Random seed" or "Open Saved Game", as you like.
<img width="1440" alt="Screen Shot 2021-08-14 at 5 16 10 PM" src="https://user-images.githubusercontent.com/87171700/129441283-cc2b79e8-d67f-4db7-9e34-70c31f814047.png">

**Comment by Jay59376** [comment]
> Well, it seems that you didn't create a world yet.
> 
> To check the uncreated world, you can click "File" on the menubar (just beside the Apple icon and Amidst name), then click "New From Seed" or "New From Random seed" or "Open Saved Game", as you like.
> 
> Like this:
> 
> <img alt="Screen Shot 2021-08-14 at 5 16 10 PM" width="1440" src="https://user-images.githubusercontent.com/87171700/129441283-cc2b79e8-d67f-4db7-9e34-70c31f814047.png">

 i want it to find the seed. So how do i do that ??????????????

**Comment by Chen-Rain** [comment]
If you have already had a world saved, you can click that "Open saved game" button, then open your save folder and find the "level.dat" file. Double click it. If you can open your world successfully, you will see your seed in the upper left corner.

by the way, the game dir is: 

```
/Users/YourUsername/Library/Application Support/minecraft/saves
```

**Comment by Jay59376** [comment]
nope, I haven't saved the world yet. How do I save it , after saving i thing i can open it.
I don't know how to save it.

**Comment by Chen-Rain** [comment]
emmm, that's weird.

I guess you wanted to check out a world's seed, am I correct? ( perhaps i misunderstood what you mean, if so, i am sorry and could you please explain it to me)

but there's a problem that you can only PLAY in your world in MINECRAFT, and you can only VIEW your world in AMIDST. The Minecraft world is stored as a folder, you may see it in the game dir.

 I think you should open Minecraft and create a world, ( or you have an existed world ). Then you will have that world in the folder I mentioned. Then you can check out the seed of your newly created world in amidst.

And, if you don't want to open Minecraft and create a new world, you can just open a new world in Amidst, however, it will be random seed unless you know the seed of the world that you like to view. What's more, the world you see in Amidst cannot store as a world folder to play in Minecraft, just view. You can copy down the seed in order to create a same world in Minecraft.

BTW, are the buttons on the menubar still working?
